Running-Mastodon/PgBouncer-guide.md

```diff
@@ -13,13 +13,18 @@ Note that you can check "PgHero" in the administration view to see how many Post
 Installing PgBouncer
 -----
 
-On Ubuntu:
+On Ubuntu/Debian:
 
     sudo apt install pgbouncer
 
-Restarting:
+Then:
 
-    sudo service pgbouncer restart
+    nano /etc/default/pgbouncer
+    START=1
```

Comment:
Ahhh! Maybe that's it. I tested it (now on 2 fresh machines) running Debian (latest stable).
I cannot get PgBouncer to start unless I change that.